.gitlab-ci.yml

```diff
@@ -169,17 +171,16 @@ variables:
   # Build images versions
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
-  DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v44534774-f5cc3e24
-  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v44534774-f5cc3e24
-  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v44534774-f5cc3e24
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v44534774-f5cc3e24
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v44534774-f5cc3e24
-
+  DATADOG_AGENT_BUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_BUILDIMAGES: v45066542-a714c2f1
+  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_WINBUILDIMAGES: v45066542-a714c2f1
+  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_ARMBUILDIMAGES: v45066542-a714c2f1
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v45066542-a714c2f1
+  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v45066542-a714c2f1
```

Comment:
You should use the images built in `main` on buildimages (you should not have the `_test_only` suffix)

Reply:
I'll update when the [buildimages PR](https://github.com/DataDog/datadog-agent-buildimages/pull/654/) is merged 👍